services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
Similar to `genres_ids`, using `list[int]` as a default argument value for `actors_ids` can lead to unexpected behavior. Consider using `None` and initializing the list inside the function.